src/extension/rn-extension.ts

```diff
@@ -8,61 +8,87 @@ import {CommandPaletteHandler} from "./commandPaletteHandler";
+                        setupReactNativeDebugger().then(() =>
+                            setupReactDir(context)), /*areErrorsFatal*/ false);
+                    entryPoint.runCode("intelliSense.setup", "Failed to setup IntelliSense", () =>
+                        IntellisenseHelper.setupReactNativeIntellisense(), /*areErrorsFatal*/ false);
+                }
+                entryPoint.runCode("debugger.setupNodeDebuggerLocation", "Failed to configure the node debugger location for the debugger", () =>
+                    configureNodeDebuggerLocation(), /*areErrorsFatal*/ false);
+                registerReactNativeCommands(context);
+            });
+    });
 }
 
 export function deactivate(): void {
     // Kill any packager processes that we spawned
-    commandPaletteHandler.stopPackager();
+    entryPoint.runCode("extension.deactivate", "Failed to stop the packager while closing React Native Tools",
+        () => commandPaletteHandler.stopPackager(), /*areErrorsFatal*/ true);
+}
+
+function configureNodeDebuggerLocation(): Q.Promise<void> {
+    const nodeDebugPath = vscode.extensions.getExtension("andreweinand.node-debug").extensionPath;
+    return fsUtil.writeFile(path.resolve(__dirname, "../", "debugger", "nodeDebugLocation.json"), JSON.stringify({ nodeDebugPath }));
+}
+
+function setupReactDir(context: vscode.ExtensionContext): Q.Promise<void> {
+    const reactDirManager = new ReactDirManager();
+    let createdDir = reactDirManager.create();
+    context.subscriptions.push(reactDirManager);
+    return createdDir;
+}
+
+function warnWhenReactNativeVersionIsNotSupported(): void {
+    return reactNativeProjectHelper.validateReactNativeVersion().done(() => { }, reason => {
+        TelemetryHelper.sendSimpleEvent("launchDebuggerError", { rnVersion: reason });
+        const shortMessage = `React Native Tools only supports React Native versions 0.19.0 and later`;
+        const longMessage = `${shortMessage}: ${reason}`;
+        vscode.window.showWarningMessage(shortMessage);
+        outputChannel.appendLine(longMessage);
+        outputChannel.show();
+    });
+}
+
+function registerReactNativeCommands(context: vscode.ExtensionContext): void {
+    // Register React Native commands
+    registerVSCodeCommand(context, "runAndroid", "Failed to run the application in Android", () => commandPaletteHandler.runAndroid());
+    registerVSCodeCommand(context, "runIos", "Failed to run the application in iOS", () => commandPaletteHandler.runIos());
+    registerVSCodeCommand(context, "startPackager", "Failed to start the React Native packager", () => commandPaletteHandler.startPackager());
+    registerVSCodeCommand(context, "stopPackager", "Failed to stop the React Native packager", () => commandPaletteHandler.stopPackager());
+}
+
+function registerVSCodeCommand(
+    context: vscode.ExtensionContext, commandName: string,
+    errorDescription: string, commandHandler: () => Q.Promise<void>): void {
```

Comment:
Thanks, I missed that! :)
Replaced line 83 from `Failed to execute command ${commandName}` to errorDescription